server/app.js

```diff
@@ -74,4 +75,6 @@ process.on('unhandledRejection', (err, promise) => {
   server.close(() => process.exit(1));
 });
 
+seedData();
```

Comment:
Not a fan of running this every time we start the app, especially since this would run for production too. Let's remove this.